game/menu/gamedisplay.gd

```diff
@@ -9,10 +9,15 @@ func setup(game_cfg: ConfigFile):
 	game_file = game_cfg
 
 	$VBoxContainer/Label.text = game_cfg.get_value("game", "name")
```

Comment:
Why isn't this line part of `update_text()`?

Reply:
this never changes

Reply:
That's a point…🤔